static/app/types/hooks.tsx

```diff
@@ -631,10 +631,8 @@ type IntegrationsFeatureGatesHook = () => {
  */
 type InviteButtonCustomizationHook = () => React.ComponentType<{
   children: (opts: {
-    /**
-     * Whether the Invite Members button is active or not
-     */
-    disabled: boolean;
+    disabledByFlag: boolean;
+    disabledBySso: boolean;
```

Comment:
I think you'll need to keep the `disabled` around for BC otherwise getsentry is going to fail no?

Reply:
Yeah, that's essentially what I was asking about [here](https://github.com/getsentry/sentry/pull/73521#issuecomment-2200739255). I'll revert the name change.